templates/start-dev-env.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+#
+# This script is to create a dev namespace on Staging environment
+#
+PROJECT_NAME=<% .Name %>
+ENVIRONMENT=stage
+ACCOUNT_ID=<% index .Params `accountId` %>
+REGION=<% index .Params `region` %>
+
+# common functions
+function usage() {
+    echo
+    echo "Usage:"
+    echo "  $0 <IAM user> <project id>"
+    echo "    - IAM user: can be found by running 'aws iam get-group --group-name ${PROJECT_NAME}-developer-${ENVIRONMENT} | jq -r .Users[].UserName'"
```

Comment:
Why not just get this from `aws sts get-caller-identity`?

Reply:
yes, and check with AWS 'developer' group.